src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
 
-**Make sure that you've enabled all the [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks) rules for your project.** They are essential and catch the most severe bugs early. The recommended [`eslint-config-react-app`](https://www.npmjs.com/package/eslint-config-react-app) preset already includes them.
+**ખાતરી કરો કે તમે તમારા પ્રોજેક્ટ માટે તમામ [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks) નિયમોને સક્ષમ કર્યા છે.** તેઓ આવશ્યક છે અને સૌથી ગંભીર ભૂલોને વહેલા પકડી લે છે. ભલામણ કરેલ [`eslint-config-react-app`](https://www.npmjs.com/package/eslint-config-react-app) પ્રીસેટમાં પહેલેથી જ તેનો સમાવેશ થાય છે.
 
-### Formatting {/*formatting*/}
+### ફોર્મેટિંગ {/*formatting*/}
 
-The last thing you want to do when sharing your code with another contributor is get into an discussion about [tabs vs spaces](https://www.google.com/search?q=tabs+vs+spaces)! Fortunately, [Prettier](https://prettier.io/) will clean up your code by reformatting it to conform to preset, configurable rules. Run Prettier, and all your tabs will be converted to spaces—and your indentation, quotes, etc will also all be changed to conform to the configuration. In the ideal setup, Prettier will run when you save your file, quickly making these edits for you.
+અન્ય યોગદાનકર્તા સાથે તમારો કોડ શેર કરતી વખતે તમે જે છેલ્લું કામ કરવા માંગો છો તે છે [tabs vs spaces](https://www.google.com/search?q=tabs+vs+spaces) વિશે ચર્ચામાં આવવું! સદનસીબે, [Prettier](https://prettier.io/) તમારા કોડને પ્રીસેટ, ગોઠવણી કરી શકાય તેવા નિયમોને અનુરૂપ થવા માટે તેને ફરીથી ફોર્મેટ કરીને સાફ કરશે. Prettier ચલાવો, અને તમારી બધી ટેબ્સ સ્પેસમાં રૂપાંતરિત થઈ જશે-અને તમારું ઇન્ડેન્ટેશન, અવતરણ વગેરે પણ ગોઠવણીને અનુરૂપ થવા માટે બદલાઈ જશે. આદર્શ સેટઅપમાં, જ્યારે તમે તમારી ફાઇલ સાચવશો ત્યારે Prettier ચાલશે, તમારા માટે આ સંપાદનો ઝડપથી કરશે.
 
-You can install the [Prettier extension in VSCode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) by following these steps:
+તમે આ પગલાંને અનુસરીને [VSCode માં Prettier એક્સટેન્શન](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) ઇન્સ્ટોલ કરી શકો છો:
 
-1. Launch VS Code
-2. Use Quick Open (press Ctrl/Cmd+P)
-3. Paste in `ext install esbenp.prettier-vscode`
-4. Press Enter
+1. VS Code લોંચ કરો
+2. ક્વિક ઓપનનો ઉપયોગ કરો (Ctrl/Cmd+P દબાવો)
+3. `ext install esbenp.prettier-vscode` માં પેસ્ટ કરો
+4. Enter દબાવો
 
-#### Formatting on save {/*formatting-on-save*/}
+#### સેવ પર ફોર્મેટિંગ {/*formatting-on-save*/}
 
-Ideally, you should format your code on every save. VS Code has settings for this!
+આદર્શ રીતે, તમારે દરેક સેવ પર તમારો કોડ ફોર્મેટ કરવો જોઈએ. VS Code આ માટે સેટિંગ્સ ધરાવે છે!
 
-1. In VS Code, press `CTRL/CMD + SHIFT + P`.
-2. Type "settings"
-3. Hit Enter
-4. In the search bar, type "format on save"
-5. Be sure the "format on save" option is ticked!
+1. VS કોડમાં, `CTRL/CMD + SHIFT + P` દબાવો.
+2. "સેટિંગ્સ" લખો
```

Comment:
I would suggest not to translate vs code keywords like "સેટિંગ્સ" because currently vs code doesn't support "Gujarati" as [locales](https://code.visualstudio.com/docs/getstarted/locales#_available-locales).